jenkins/jmx-queue-blocked.yml

```diff
@@ -0,0 +1,12 @@
+type: jmx
+
+observation:
+  - name: jenkins-queue-blocked
+    metricNamespace: jenkins
+    objectName: io.jenkins:type=Queue,name=Blocked
+
+    metric:
+      - name: queue.blocked.count
```

Comment:
Use plural `queues`? And in few other places.

Reply:
I think this is not queues in stuck, but something else stuck in queue